src/debsbom/commands/repack.py

```
@@ -96,3 +96,7 @@ def setup_parser(cls, parser):
             help="validate generated SBOM (only for SPDX)",
             action="store_true",
         )
+        parser.add_argument(
+            "--mtime",
+            help="set mtime for creating tar archives",
```

We should mention that this argument is passed to directly to `tar`, otherwise it might not be clear what time formats are allowed here.

---

See in the discussion below, it would be better if we accept only ISO 8601 strings here. We can directly parse this into a `datetime` and fail early if the passed time is bad. This is already done like this in the `generate` command with the `--timestamp` option